states/webSocketIO.py

```diff
@@ -0,0 +1,163 @@
+# state file generated using paraview version 5.9.0
+
+#### import the simple module from the paraview
+from paraview.simple import *
+#### disable automatic camera reset on 'Show'
+paraview.simple._DisableFirstRenderCameraReset()
+
+# ----------------------------------------------------------------
+# setup views used in the visualization
+# ----------------------------------------------------------------
+
+# get the material library
+materialLibrary1 = GetMaterialLibrary()
+
+# Create a new 'Render View'
+renderView1 = CreateView('RenderView')
+renderView1.ViewSize = [678, 402]
+renderView1.AxesGrid = 'GridAxes3DActor'
+renderView1.StereoType = 'Crystal Eyes'
+renderView1.CameraPosition = [0.055230719568613376, 0.9258039835776175, 5.617192129740201]
+renderView1.CameraFocalPoint = [0.028925400103161122, -0.07452783924491535, 0.013167622178124871]
+renderView1.CameraViewUp = [0.4851148099218602, 0.8604466851591295, -0.1558689295310115]
+renderView1.CameraFocalDisk = 1.0
+renderView1.CameraParallelScale = 1.473370383194758
+renderView1.BackEnd = 'OSPRay raycaster'
+renderView1.OSPRayMaterialLibrary = materialLibrary1
+
+# Create a new 'Render View'
+renderView2 = CreateView('RenderView')
+renderView2.ViewSize = [677, 402]
+renderView2.AxesGrid = 'GridAxes3DActor'
+renderView2.StereoType = 'Crystal Eyes'
+renderView2.CameraPosition = [-2.1956880750963, 1.1012529795769979, 2.8984371371228597]
+renderView2.CameraFocalPoint = [-0.10797200872540381, 0.36903090897439744, -0.16194931005432628]
+renderView2.CameraViewUp = [-0.0374390542532092, 0.9657889167252136, -0.25661271509292716]
+renderView2.CameraFocalDisk = 1.0
+renderView2.BackEnd = 'OSPRay raycaster'
+renderView2.OSPRayMaterialLibrary = materialLibrary1
+
+SetActiveView(None)
+
+# ----------------------------------------------------------------
+# setup view layouts
+# ----------------------------------------------------------------
+
+# create new layout object 'Layout #1'
+layout1 = CreateLayout(name='Layout #1')
+layout1.SplitHorizontal(0, 0.500000)
+layout1.AssignView(1, renderView1)
+layout1.AssignView(2, renderView2)
+layout1.SetSize(1356, 402)
+
+# ----------------------------------------------------------------
+# restore active view
+SetActiveView(renderView1)
+# ----------------------------------------------------------------
+
+# ----------------------------------------------------------------
+# setup the data processing pipelines
+# ----------------------------------------------------------------
+
+ttkIcosphere1 = TTKIcosphere(registrationName='TTKIcosphere1')
+ttkIdentifiers1 = TTKIdentifiers(registrationName='TTKIdentifiers1', Input=ttkIcosphere1)
```

Comment:
TTKIdentifiers will be marked deprecated any time soon.
please use the native "GenerateIds" instead.